### PR TITLE
Changes to title and H1

### PR DESCRIPTION
**Pull request by walsehgal**
Changes to title and H1 to make them more relevant

### CODE REVIEW

**Comment by walsehgal** [comment]
@traya1

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Tutorial-Audio-Conferencing.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ### YamlMime:Tutorial
-title: Audio Conferencing in Microsoft Teams
+title: 'Tutotial: Audio Conferencing - Microsoft Teams'
 metadata:
-  title: Audio Conferencing in Microsoft Teams
+  title: 'Tutorial: Getting started with audio conferencing in Microsoft Teams'
```
Comment (JohanFreelancer9) [inline Teams/Tutorial-Audio-Conferencing.yml]:
```suggestion
  title: 'Tutorial: Getting started with Audio Conferencing in Microsoft Teams'
```

File: Teams/audio-conferencing-subscription-dial-out.md [hunk 2]
```diff
@@ -20,10 +20,10 @@ f1.keywords:
 - CSH
 ms.custom: 
   - Licensing
-description: "Customers have been provided a complimentary dial-out capability that will end on November 30, 2019.  Beginning December 1, 2019, each audio conferencing subscription provides 60 minutes per user per month to ANY of the Zone A countries as described in this document. "
+description: Dial-Out and Call Me At minutes benefit. As of December 1, 2019 each audio conferencing subscription provides 60 minutes per-user per-month to Zone A countries.
```
Comment (JohanFreelancer9) [inline Teams/audio-conferencing-subscription-dial-out.md]:
```suggestion
description: Dial-Out and Call Me At minutes benefit. As of December 1, 2019, each audio conferencing subscription provides 60 minutes per-user per-month to Zone A countries.
```

File: Teams/audio-conferencing-subscription-dial-out.md [hunk 3]
```diff
@@ -33,9 +33,9 @@ Customers have been provided a [complimentary dial-out capability](complimentary
 > The end of the complimentary dial-out period will not take place on November 30,2019 for countries where the Audio Conferencing subscription is available, but we do not currently provide the ability to set up Communications Credits. These specific countries are Russia, South Korea, and Taiwan.
```
Comment (JohanFreelancer9) [inline Teams/audio-conferencing-subscription-dial-out.md]:
```suggestion
> The end of the complimentary dial-out period will not take place on November 30, 2019, for countries where the Audio Conferencing subscription is available, but we do not currently provide the ability to set up Communications Credits. These specific countries are Russia, South Korea, and Taiwan.
```

File: Teams/audio-conferencing-subscription-dial-out.md [hunk 4]
```diff
@@ -63,8 +63,8 @@ Consider the following example. A customer has purchased 115 Audio Conferencing
 
 ## How can I monitor minute pool usage?
 
-- You can monitor the usage against your dial-out minute pool in the “legacy” Skype for Business Admin Center. In the Microsoft Teams Admin Center, navigate to **Legacy portal** > **Reports** > **PSTN Minute Pools**. The Zone A dial-out minute pool will be labeled in the report as “Outbound Calls to Zone A Countries.”
-- Email notifications will be sent to all tenant administrators of a given customer when the utilization of the tenant’s dial-out minutes pool has reached 80% and 100%.
+- You can monitor the usage against your dial-out minute pool in the "legacy" Skype for Business Admin Center. In the Microsoft Teams Admin Center, navigate to **Legacy portal** > **Reports** > **PSTN Minute Pools**. The Zone A dial-out minute pool will be labeled in the report as "Outbound Calls to Zone A Countries."
```
Comment (JohanFreelancer9) [inline Teams/audio-conferencing-subscription-dial-out.md]:
```suggestion
- You can monitor the usage against your dial-out minute pool in the "legacy" Skype for Business Admin Center. In the Microsoft Teams Admin Center, navigate to **Legacy portal** > **Reports** > **PSTN Minute Pools**. The Zone A dial-out minute pool will be labeled in the report as "Outbound Calls to Zone A Countries".
```

File: Teams/change-the-settings-for-an-audio-conferencing-bridge.md [hunk 5]
```diff
@@ -22,7 +22,7 @@ f1.keywords:
 ms.custom: 
   - Audio Conferencing
   - ms.teamsadmincenter.audioconferencing.bridgesettings
-description: "Get the steps you need to change settings for a conferencing bridge that's used to prompt callers and gather names and pins for meeting organizers when they're not using Skype for Business or Microsoft Teams apps. "
+description: Change audio conferencing bridge settings including entry and exit notifications, play names or phone numbers, tones, and prompt callers to record their name.
```
Comment (JohanFreelancer9) [inline Teams/change-the-settings-for-an-audio-conferencing-bridge.md]:
```suggestion
description: Change audio conferencing bridge settings, including entry and exit notifications, play names or phone numbers, tones, and prompt callers to record their name.
```

File: Teams/code-of-practice-for-the-united-kingdom-u-k.md [hunk 6]
```diff
@@ -21,7 +21,7 @@ f1.keywords:
 ms.custom: 
   - Legal
 hideEdit: true
-description: "Learn the code of practice for Calling Plans (PSTN Calling plans)  for Skype for Business for the United Kingdom, where to get help, and how to submit complaints and resolve disputes. "
+description: Learn the code of practice for PSTN Calling plans for Skype for Business UK including where to get help, and how to submit complaints and resolve disputes.
```
Comment (JohanFreelancer9) [inline Teams/code-of-practice-for-the-united-kingdom-u-k.md]:
```suggestion
description: Learn the code of practice for PSTN Calling plans for Skype for Business UK including, where to get help, and how to submit complaints and resolve disputes. 
```

File: Teams/coexistence-chat-calls-presence.md [hunk 7]
```diff
@@ -16,7 +16,7 @@ ms.collection:
   - M365-collaboration
 appliesto: 
   - Microsoft Teams
-description: "This document describes the behavior of chat, call routing and presence between users of Teams and Skype for Business, both in-tenant and federated, based on assigned TeamsUpgrade modes. It includes routing optimizations,  presence behavior, as well as the change of default TeamsUpgrade mode from *Legacy* to *Islands* and the impending retirement of *Legacy*."
+description: Coexistence behavior between Teams & Skype for Business including routing parameters, chat & call routing, chats & calls from pre-existing threads, & presence.
```
Comment (JohanFreelancer9) [inline Teams/coexistence-chat-calls-presence.md]:
```suggestion
description: Coexistence behavior between Teams & Skype for Business, including routing parameters, chat & call routing, chats & calls from pre-existing threads, & presence.
```

File: Teams/complimentary-dial-out-period.md [hunk 8]
```diff
@@ -52,9 +52,9 @@ Users with a license usage location (this is the user country location defined i
 - All outbound calls to destinations not in the Zone A country list are billed per minute using Communications Credits at our published rates to that destination (provided tenant has set up Communications Credits and assigned the Communications Credits license to the meeting organizer).
 
 > [!NOTE]
-> You can monitor the usage against dial-out minute pool in the Skype for Business Admin Center. In the Microsoft Teams & Skype Admin Center, go to **Legacy portal** > **Reports** > **PSTN Minute Pools**. This complimentary minute pool will be labeled in the report as “Outbound Calls to Zone A countries and regions.”
+> You can monitor the usage against dial-out minute pool in the Skype for Business Admin Center. In the Microsoft Teams & Skype Admin Center, go to **Legacy portal** > **Reports** > **PSTN Minute Pools**. This complimentary minute pool will be labeled in the report as "Outbound Calls to Zone A countries and regions."
```
Comment (JohanFreelancer9) [inline Teams/complimentary-dial-out-period.md]:
```suggestion
> You can monitor the usage against dial-out minute pool in the Skype for Business Admin Center. In the Microsoft Teams & Skype Admin Center, go to **Legacy portal** > **Reports** > **PSTN Minute Pools**. This complimentary minute pool will be labeled in the report as "Outbound Calls to Zone A countries and regions".
```

File: Teams/enable-features-office-365.md [hunk 9]
```diff
@@ -8,7 +8,7 @@ ms.service: msteams
 audience: admin
 ms.reviewer: ritikag
 search.appverid: MET150
-description: Learn how to turn on or off Microsoft Teams org-wide settings for your organization, including apps, external access, guest access, Teams settings, and Teams upgrade preferences.
+description: Learn how to turn on or off Microsoft Teams org-wide settings including apps, external access, guest access, Teams settings, and Teams upgrade preferences.
```
Comment (JohanFreelancer9) [inline Teams/enable-features-office-365.md]:
```suggestion
description: Learn how to turn on or off Microsoft Teams org-wide settings, including apps, external access, guest access, Teams settings, and Teams upgrade preferences.
```

File: Teams/manage-the-audio-conferencing-settings-for-a-user-in-teams.md [hunk 10]
```diff
@@ -20,7 +20,7 @@ f1.keywords:
 - CSH
 ms.custom: 
   - Audio Conferencing
-description: "As an Office 365 admin, you can edit the Microsoft Teams Audio Conferencing settings—such as the provider, default toll or toll-free number, conference ID, or PIN—for an individual user in your organization. "
+description: An Office 365 admin can edit the Teams Audio Conferencing settings including provider, default toll or toll-free number, conference ID, or PIN for a user.
```
Comment (JohanFreelancer9) [inline Teams/manage-the-audio-conferencing-settings-for-a-user-in-teams.md]:
```suggestion
description: An Office 365 admin can edit the Teams Audio Conferencing settings, including provider, default toll or toll-free number, conference ID, or PIN for a user.
```

File: Teams/plan-Office-365-groups.md [hunk 11]
```diff
@@ -8,7 +8,7 @@ ms.date: 08/29/2018
 ms.topic: conceptual
 ms.service: msteams
 audience: admin
-description: Learn the decisions you should make when planning for Office 365 Groups such as choosing Public and Private groups, using the Teams client or the Office 365 admin web console, and how to educate your teams about using conversations.
+description: Learn about planning for Office 365 Groups in Teams including the differences between Groups & Teams conversations, and how Teams respects Groups naming policy.
```
Comment (JohanFreelancer9) [inline Teams/plan-Office-365-groups.md]:
```suggestion
description: Learn about planning for Office 365 Groups in Teams, including the differences between Groups & Teams conversations, and how Teams respects Groups naming policy.
```

File: Teams/prepare-network.md [hunk 12]
```diff
@@ -7,7 +7,7 @@ ms.topic: article
 ms.service: msteams
 ms.reviewer: jastark, kojika
 audience: admin
-description: Before you roll out Microsoft Teams, evaluate and prepare your network to be sure it's ready for Teams. Information includes network requirements, bandwidth requirements, and network optimization guidance.
+description: Learn about preparing your organization's network for Microsoft Teams including network requirements, network optimization, and bandwidth requirements.
```
Comment (JohanFreelancer9) [inline Teams/prepare-network.md]:
```suggestion
description: Learn about preparing your organization's network for Microsoft Teams, including network requirements, network optimization, and bandwidth requirements.
```

File: Teams/set-up-calling-plans.md [hunk 13]
```diff
@@ -21,7 +21,7 @@ f1.keywords:
 ms.custom: 
   - Calling Plans
   - LIL_Placement
-description: "For Office 365 Calling Plan, learn how to buy and set up licenses, get phone numbers, add and assign emergency locations and phone numbers to users, and tell your users about their new phone numbers."
+description: Learn to set up Calling Plans including view plans available in your region, buy & assign licenses, get phone numbers, and add emergency addresses & locations.
```
Comment (JohanFreelancer9) [inline Teams/set-up-calling-plans.md]:
```suggestion
description: Learn to set up Calling Plans, including view plans available in your region, buy & assign licenses, get phone numbers, and add emergency addresses & locations.
```

File: Teams/toll-free-dialing-limitations-and-restrictions.md [hunk 14]
```diff
@@ -21,7 +21,7 @@ f1.keywords:
 ms.custom: 
   - Calling Plans
   - Strat_SB_PSTN
-description: "This will help you find out how to dial toll free numbers in each country/region. After you select the country/region, it will take you to a country-specific page that contains specific details, restrictions, and limits for toll-free service availability where toll-free service is available. The dialing format or formats will show you the required access codes within each country/region to dial the toll-free number."
+description: How to dial toll free numbers in each country/region including details, restrictions, limits for toll-free service availability, and required access codes.
```
Comment (JohanFreelancer9) [inline Teams/toll-free-dialing-limitations-and-restrictions.md]:
```suggestion
description: How to dial toll-free numbers in each country/region, including details, restrictions, limits for toll-free service availability, and required access codes.
```

File: Teams/upgrade-to-Teams-execute-SkypeforBusinessHybridOnPrem.md [hunk 15]
```diff
@@ -8,7 +8,7 @@ ms.topic: article
 ms.service: msteams
 audience: admin
 ms.reviewer: bjwhalen
-description: Considerations for upgrading to Teams from a Skype for Business on-premises deployment.  
+description: Deploying and implementing your upgrade to Teams from a Skype for Business Server or Microsoft Lync on-premises deployment.
```
Comment (JohanFreelancer9) [inline Teams/upgrade-to-Teams-execute-SkypeforBusinessHybridOnPrem.md]:
```suggestion
description: Deploying and implementing your upgrade to Teams from a Skype for Business server or Microsoft Lync on-premises deployment.  
```

**Pull request closed without merging**